### ISSUE TITLE
Transcripts complete in SM not coming back to PUA

### ISSUE THREAD

**Issue by annewootton** [popuparchive/pop-up-archive#1009]
Or at least, not on the first try. Good news: customers expect faster, because usually we are.

Here's the current culprit: https://www.popuparchive.com/collections/4010/items/34793

I can nudge it along manually but wanted to document here first. We just need it by tomorrow.

**Comment by annewootton** [comment]
Looks like it finished on its own... hmm. Leaving this open for now since it's happened multiple times — maybe our nudger logic should be more aggressive?

**Comment by pkarman** [comment]
the nudger work window is currently 4 hours, which is the max time a task will be tried before it is cancelled as incomplete.

In this case the speechmatics callback did fire at 2015-02-17 21:16:03.695029+00:00 but I had neglected to run the 'rake db:migrate' task yesterday when I deployed the new branch and so the callback worker had failed. 

I did subsequently run that task after the callback failed, so when the nudger ran 4 hours later it successfully completed the task.

We can make the work window smaller if we think that's useful. I had been waiting until we owned our own fixer instance before tweeking the timings, just so the inevitable knob-adjustments were all under our own supervision.
